student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './person'
+
+class Student < Person
+  def initialize(age, classroom, _name = 'Unknown', _parent_permission: true)
+    super(age, _name = 'Unknown', parent_permission: true)
```

Comment:
- [ ] As requested by the previous reviewer, kindly update this as `parent_permission: parent_permission`. What you are doing is always setting the `parent_permission: true` which in case it is set to false will not work. Kindly fix this issue.